### PR TITLE
Dockerfiles and associated notes

### PR DESCRIPTION
**Pull request by marrobi**
Cellbase docker implementation. Steps to build using docker, create docker image with the build in, then one with tomcat too.

### CODE REVIEW

File: cellbase-app/app/docker/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Building and running Cellbase in Docker
+
+## Build Cellbase
+
+To build cellbase run the follwoing command fromt he root of the repository:
```
Comment (martinpeck) [inline cellbase-app/app/docker/README.md]: typo 'fromt'

File: cellbase-app/app/docker/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Adds tomcat and healthcheck.
+
+```
+docker build -t cellbase-app  ./cellbase-app/app/docker/cellbase-app/
+```   
+
+## Run Cellbase server
+
+The command below runs cellbase and mounts configuration.json from the specified path. This alows configuration to be persisted outside of the container and also shared between containers.
```
Comment (martinpeck) [inline cellbase-app/app/docker/README.md]: typo `alows`

File: cellbase-app/app/docker/README.md (hunk 1 above)
Comment (marrobi) [reply cellbase-app/app/docker/README.md]: I've not proof read anything, feel free to pick the PR up. Just wanted to get it in there as not going to make progress this week. Feel free to fix the notes up. I actually thought draft would prevent from reviewing it.

Comment (martinpeck) [reply cellbase-app/app/docker/README.md]: Nope. Prevents anyone from merging it.

**Comment by marrobi** [comment]
These images have been tested and I believe work correctly. The build could do with being integrated into CI as with IVA/OpenCGA images.

Will create an issue to track request for environment variable configuration support to remove need to mount configuration.json.

File: cellbase-app/app/docker/cellbase-app/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      org.label-schema.build-date=${BUILD_DATE} \
+      org.label-schema.docker.schema-version="1.0"
+
+# Install local dependencies
+RUN apt-get update && apt-get install -y wget tar \
+  && rm -rf /var/lib/apt/lists/* \
+  && apt-get clean
+
+# Download and install Tomcat
+RUN wget --quiet --no-cookies http://www-eu.apache.org/dist/tomcat/tomcat-8/v${TOMCAT_VERSION}/bin/apache-tomcat-${TOMCAT_VERSION}.tar.gz -O /tmp/tomcat.tgz && \
```
Comment (j-coll) [inline cellbase-app/app/docker/cellbase-app/Dockerfile]: Same as in https://github.com/opencb/opencga/pull/1216 , the URL used to download Tomcat will fail as soon as a new version is deployed. We should use the archive URL, or download from the repositories, with `apt-get`

Comment (marrobi) [reply cellbase-app/app/docker/cellbase-app/Dockerfile]: done.

**Pull request merged**